docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+rabbitmq:
+    image: "rabbitmq:3-management"
+    hostname: "my-rabbit"
+    labels:
+        name: "some-rabbit"
+    ports:
+    - "15672:15672"
+    - "3131:5672"
```

Comment:
Any reason to use different virtual port?

Reply:
Yes, in the expectations of the task says that we should use a non-default port with rabbitmq.